src/main/resources/templates/blog.html

```diff
@@ -1,60 +1,54 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Blog</title>
-    <link rel="stylesheet" href="style.css">
-</head>
-<body>
-    <header>
-        <nav>
-            <ul>
-                <li><a href="index.html">Home</a></li>
-                <li><a href="blog.html">Blog</a></li>
-                <li><a href="signup.html">Sign Up</a></li>
-            </ul>
-        </nav>
-    </header>

```

Comment:
Reusable html code that is used on multiple pages can be implemented using fragments instead of rewriting long pieces of code again and again. If you need to update the navigation component in the future, having the code in one place will save time and reduce the chances of missing the update if you had to update every page individually. See [Thymeleaf Fragments](https://www.thymeleaf.org/doc/articles/layouts.html#fragment-expressions).